<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKSaveOutputDataSettings</class>
 <widget class="QDialog" name="SAKSaveOutputDataSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据保存设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>文件设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="pathLineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="setFilePushButton">
        <property name="text">
         <string>设置文件</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>输出文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>格式设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QRadioButton" name="hexRadioButton">
        <property name="text">
         <string>以十六进制文本格式保存至文件</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="binRadioButton">
        <property name="text">
         <string>以二进制格式保存至文件</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="utf8RadioButton">
        <property name="text">
         <string>以UFT8文本格式保存至文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="closePushButton">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="clearFilePushButton">
     <property name="text">
      <string>清空文件</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2sssss">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#label_2sssss {
    color:red
}</string>
     </property>
     <property name="text">
      <string>只保存接收到的数据</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closePushButton</sender>
   <signal>clicked()</signal>
   <receiver>SAKSaveOutputDataSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
